javascript/javascript1/week3/Note_taking/notes.js

```diff
@@ -0,0 +1,35 @@
+const notes = [];
+function addNote(content, id) {
+    notes.push({content:content,id:id});
```

Comment:
technically legal, but confusing. I would name the function arguments something else like `content_in` and `id_in`

Reply:
https://github.com/Aswini-D3/hyf-homework/blob/javascript/javascript1/week3/javascript/javascript1/week3/seriesTime.js
I changed the code for 80 years, please check once.